packages/shared/src/components/modals/post/CreateSharedPostModal.tsx

```diff
@@ -151,32 +136,18 @@ export function CreateSharedPostModal({
           </Switch>
         )}
       </form>
+      <span className="flex flex-row items-center gap-2 px-4">{footer}</span>
       <Modal.Footer className="typo-caption1" justify={Justify.Start}>
-        <Button
-          icon={<AtIcon />}
-          className="btn-tertiary"
-          size={ButtonSize.Small}
-          onClick={markdownRef?.current?.onMentionCommand}
-        />
-        <Divider vertical />
-        <SourceButton source={squad} size="small" />
-        <span className="-ml-1 flex-1">
-          <strong>{squad.name}</strong>
-          <span className="ml-1 text-theme-label-tertiary">
-            @{squad.handle}
-          </span>
-        </span>
+        {footer}
 
-        {!isMobile && (
-          <Button
-            className="btn-primary-cabbage ml-auto"
-            disabled={isPosting}
-            loading={isPosting}
-            form="share_post"
-          >
-            Post
-          </Button>
-        )}
+        <Button
```

Comment:
Doesn't this need isMobile anymore?

Reply:
Not anymore, similar to Drawers, Modal.Footer is not rendered on Modal Forms since these action buttons are now part of the header.